tests/php/test-fieldmanager-datasource-term.php

```diff
@@ -348,4 +348,38 @@ public function test_taxonomy_made_sortable() {
 		$this->assertTrue( $wp_taxonomies['category']->sort );
 		$this->assertTrue( $wp_taxonomies['post_tag']->sort );
 	}
+
+	public function test_append_true_after_first_save() {
+		$fm = new Fieldmanager_Group( array(
+			'name'     => 'author_data',
+			'children' => array(
+				'school' => new Fieldmanager_Group(array(
+					'label' => 'School',
+					'add_more_label' => 'Add new',
+					'limit' => 0,
+					'children' => array(
+						'school_city' => new Fieldmanager_Autocomplete( array(
+							'label' => 'School city',
+							'datasource' => new Fieldmanager_Datasource_Term( array(
+								'taxonomy' => 'post_tag',
+								'taxonomy_save_to_terms' => true
+							))
+						)),
+					)
+				))
+			)
```

Comment:
Ideally the above code would adhere to WP Coding Standards (commas after last elements, spaces between `))`)